platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/NativeMapView.java

```diff
@@ -381,14 +381,14 @@ public void setZoom(double zoom) {
     if (isDestroyedOn("setZoom")) {
       return;
     }
-    setZoom(zoom, 0);
+    setZoom(zoom, -1, -1, 0);
```

Comment:
What do you think of replacing those magic numbers with a symbolic constant in order to increase readability?
Magic numbers are numbers with special values that usually are not obvious.
If the numbers might ever change, making the change is a nightmare.